src/Kaleido.jl

```diff
@@ -1,11 +1,14 @@
 module Kaleido
 
-import JSON
+using JSON: JSON
 using Base64
 using Kaleido_jll
 
-export savefig
+export save
```

Comment:
Is that defined somewhere? Should we export `savefig` and `save_payload` here or rather not?

Reply:
I just changed it to export `savefig`.  I'll leave `save_payload` unexported since it's fairly low-level